apps/mobile/src/queries/balance/runes-balance.query.ts

```diff
@@ -0,0 +1,47 @@
+import { useTotalBitcoinAccountServiceRequests } from '@/hooks/use-bitcoin-account-service-requests';
+import { toFetchState } from '@/shared/fetch-state';
+import { QueryFunctionContext, useQuery } from '@tanstack/react-query';
+
+import { BitcoinAccountIdentifier, getRunesBalancesService } from '@leather.io/services';
+
+export function useRunesTotalBalance() {
+  // TODO LEA-1982: check with Alex once we have xpub-based runes endpoint from BIS
```

Comment:
@alexp3y I just stubbed out this `Runes` code, we can revisit it once the BIS endpoint is ready and probably refactor this